importer/celery_tasks.py

```diff
@@ -1,25 +1,27 @@
 import logging
-import os
 from typing import Optional
 from uuid import UUID
 
 from celery import Task
 from django.utils import timezone
 from django.utils.module_loading import import_string
-
+from django.utils.translation import ugettext
 from importer.api.exception import (InvalidInputFileException,
-                                    PublishResourceException,
+                                    PublishResourceException, ResourceCopyException,
                                     ResourceCreationException,
                                     StartImportException)
```

Comment:
We may want to use consistent naming:
e.g. ACTIONResourceException or ResourceACTIONException (not a mix of the two)